src/lib/utils/isCloseToEdge.ts

```diff
@@ -0,0 +1,53 @@
+import { isFunction } from "lodash"
+import { NativeScrollEvent } from "react-native"
+
+/**
+ * Detects if a <Flatlist /> has scrolled to the edge of his loaded content. Taken from from RN’s ListView.js
+ *
+ *  @example
+ *
+ *  handleScroll = isCloseToEdge(() => {
+ *    this.loadMore()
+ *  })
+ *
+ *  return (
+ *    <Flatlist horizontal onScroll={handleScroll}>
+ *      <Artworks />
+ *    </Flatlist>
+ *  )
+ */
+
+const PAGE_END_THRESHOLD = 1000
+
+type CallBack = () => void
+
+export function isCloseToEdge(onScrollEnd: CallBack, pageEndThreshold: number = PAGE_END_THRESHOLD) {
```

Comment:
Are we planning to add tests for this class? I'm comfortable merging as-is as long as we ticket the follow-up tests.

Reply:
We can, looks like there are no tests yet for `isCloseToBottom` so they could be grouped together.

Reply:
Added a ticket: https://artsyproduct.atlassian.net/browse/ME-99